releasenotes/notes/improve-qv-plotter-061c1b362c28a076.yaml

```diff
@@ -0,0 +1,13 @@
+---
+fixes:
+  - |
+    Improve :meth:`qiskit.ignis.verification.quantum_volume.QVFitter.plot_qv_data`.
+    * Change Y axis label to "heavy output probability".
+    * Set figure default title inside the method rather than setting outside like the tutorial and textbook chapter.
+    * Change error bar to 2-sigma (success confidence interval requirement) instead of 1-sigma.
+    * Add caps to error bars to increase readbility when ideal and experimental values overlaps.
+    * Use different markers for ideal and experimental values.
+    * Return `matplotlib.Figure` similar to :meth:`qiskit.visualization.plot_histogram` in Qiskit Terra..
+
+
+
```

Comment:
Can we rework this a bit so it'll render a bit more clearly when the release goes out:

```suggestion
features:
  - |
    The :class:`qiskit.ignis.verification.QVFitter` method
    :meth:`~qiskit.ignis.verification.QVFitter.plot_qv_data` has been updated to return a
    ``matplotlib.Figure`` object. Previously, it would not return anything. By returning a figure
     this makes it easier to integrate the visualizations into a larger ``matplotlib`` workflow.
  - |
    A new kwarg, ``figsize`` has been added to the
    :class:`qiskit.ignis.verification.QVFitter` method
    :meth:`qiskit.ignis.verification.QVFitter.plot_qv_data`. This kwarg takes in a tuple of the
    form ``(x, y)`` where ``x`` and ``y`` are the dimension in inches to make the generated
    plot.
```

I left out the other changes here, because those 2 notes are the big user facing API changes to the function. Normally for release notes we don't need to highlight every change made in a PR, just the ones with user facing impact. Things that we want to tell our users when we're upgrading from the old version to the new one. That being said I'm not a big user of `plot_qv_data`, so I  probably am just missing how significant they are, so please feel free to keep any you view as having important user facing significance.

Another thing I should point out is that for release notes you shouldn't assume a shared context. The reader of the
release notes is an ignis user, but may not be familiar with its internals or the wider Qiskit project. For example,
things like "rather than setting outside like the tutorial and textbook chapter." don't really translate because it assumes
a shared context about the qiskit tutorial's content and the qiskit textbook chapter on QV. It might also be useful to add a `jupyter-execute` to generate a visualization to show the changes too.

Reply:
Thanks for the suggestions. I will take a fix the release note later today and add jupyter-execute.

Reply:
I added error bar changes to the release note because I think it's very important for users to know. I also added `(2 sigma error)` to the legend to explicitly shows what the error bars represent.

Reply:
I am not sure `jupyter-execute` is a good idea because the data for plotting requires many lines of code to generate.